speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/DecisionMaker.inl

```diff
@@ -3,11 +3,12 @@
 #include "DecisionTuple.h"
 #include "Mediator.h"
 #include "ConfigEnums.h"
+#include "../rppUtils/RppUtils.hpp"
 
 
 /// @brief  Creates an implementation of a decision maker
 #define CREATE_DECISION_MAKER_IMPLEMENTATION(type1,type2) \
-    template void DecisionMaker<type1,type2>::Initialize(tCarElt* p_initialCar, tSituation* p_initialSituation, BlackBoxData* p_testSituations, int p_testAmount);\
+    template void DecisionMaker<type1,type2>::Initialize(tCarElt* p_initialCar, tSituation* p_initialSituation, const std::string& p_blackBoxExecutablePath, BlackBoxData* p_testSituations, int p_testAmount);\
```

Comment:
This does go far off to the right at character 209, while i think our coding standard was something like, at most to charachter 120